harmonica/tests/test_tesseroid_layer.py

```diff
@@ -77,6 +77,47 @@ def tesseroid_layer_with_holes(dummy_layer):
     return tesseroids, density
 
 
+@pytest.mark.parametrize(
+    "longitude_w, longitude_e",
+    [
+        (0, 480),
+        (0, 360),
+        (-180, 180),
+        (0, 360 - 18.94736842 / 2),
+    ],
+)
+def test_tesseroid_overlap_wrong_coords(longitude_w, longitude_e, mean_earth_radius):
+    """
+    Check if the tesseroid are overlapped
+    """
+    latitude = np.linspace(-10, 10, 6)
+    longitude = np.linspace(longitude_w, longitude_e, 20)
```

Comment:
That float with so many decimal figures is not looking good. If you create the longitude array with 21 elements:
```python
longitude = np.linspace(longitude_w, longitude_e, 21)
```

then, the spacing will be of 18 degrees. And instead of testing with 18 / 2 you can just put 18 - 0.01. Those coordinates should be caught as invalid even with the small difference.

Reply:
Thank you for the suggestion. I will make the change.